_includes/connect-zwa-2/migrate-z-wave-network.md

```diff
@@ -0,0 +1,52 @@
+<!--- more generic steps on migrating a Z-Wave network to Home Assistant Connect ZWA-2. -->
+
+
+{% steps %}
+{% include 'connect-zwa-2/assemble-and-install.md' %}
+
+
+{% step "You have the Z-Wave JS add-on" "Scenario 1" %}
+{% image "/static/img/connect-zwa-2/zwa-2-z-wave-js-add-on.png" "Screenshot of Z-Wave JS add-on" %}
+{% stepContent %}
+
+**Scenario 1**: You already have a Z-Wave network running with the **Z-Wave JS** add-on:
+
+- Follow the [steps in the Z-Wave integration documentation on migrating your Z-Wave network to Home Assistant Connect ZWA-2](https://www.home-assistant.io/integrations/zwave_js/#migrating-a-z-wave-network-to-a-new-adapter).
+- **Troubleshooting**: If you don't know whether you use the **Z-Wave JS** add-on:
+  - Go to **Settings** > **Add-ons** and look for the **Z-Wave JS** add-on.
```

Comment:
As this is a core add-on we can use a my-link here to go directly to the add-on page.

Reply:
Thank you for the catch! https://github.com/NabuCasa/support/pull/641